File: maple/maple-user-web/src/main/webapp/js/service/jobService.js
//服务层
app.service('jobService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../job/findAll.do');		
	}
	//分页 
	this.findPage=function(page,rows){
		return $http.get('../job/findPage.do?page='+page+'&rows='+rows);
	}
	//查询实体
	this.findOne=function(id){
		return $http.get('../job/findOne.do?id='+id);
	}
	//增加 
	this.add=function(entity){
		return  $http.post('../job/add.do',entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.post('../job/update.do',entity );
	}
	//删除
	this.dele=function(ids){
		return $http.get('../job/delete.do?ids='+ids);
	}
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('../job/search.do?page='+page+"&rows="+rows, searchEntity);
	}    	
});

File: maple/maple-type-interface/src/main/java/com/maple/service/JobService.java
package com.maple.service;
import java.util.List;

import com.maple.pojo.Job;

import entity.PageResult;
/**
 * 服务层接口
 * @author Administrator
 *
 */
public interface JobService {

	/**
	 * 返回全部列表
	 * @return
	 */
	public List<Job> findAll();
	
	
	/**
	 * 返回分页列表
	 * @return
	 */
	public PageResult findPage(int pageNum, int pageSize);
	
	
	/**
	 * 增加
	*/
	public void add(Job job);
	
	
	/**
	 * 修改
	 */
	public void update(Job job);
	

	/**
	 * 根据ID获取实体
	 * @param id
	 * @return
	 */
	public Job findOne(int id);
	
	
	/**
	 * 批量删除
	 * @param ids
	 */
	public void delete(int[] ids);

	/**
	 * 分页
	 * @param pageNum 当前页 码
	 * @param pageSize 每页记录数
	 * @return
	 */
	public PageResult findPage(Job job, int pageNum, int pageSize,String name);

	public List<Job> findAll(String name);
}
